apps/modernization-ui/src/apps/page-builder/pages/BusinessRulesLibrary/BusinessRulesLibrary.tsx

```diff
@@ -38,28 +38,33 @@ export const BusinessRulesLibrary = ({ modalRef }: any) => {
     }, [searchQuery, currentPage, pageSize, sortBy, filter, page]);
 
     return (
-        <PageBuilder page="pages">
-            <div className="business-rules-library padding-top-3">
-                {page && (
-                    <div className="padding-left-3">
-                        <Breadcrumb
-                            start="../.."
-                            currentPage="Rules library"
-                            crumbs={[{ name: page.name, position: 1, to: '..' }]}>
-                            Page library
-                        </Breadcrumb>
-                    </div>
-                )}
-                <div className="business-rules-library__container">
-                    <div className="business-rules-library__table">
-                        <BusinessRulesLibraryTable
-                            summaries={rules}
-                            qtnModalRef={modalRef}
-                            pages={{ currentPage, pageSize, totalElements }}
-                        />
+        <>
+            <header className="business-rule-header">
+                <h2>Page builder</h2>
+            </header>
+            <PageBuilder page="pages">
```

Comment:
Suggestion: I would recommend removing the use of the `<PageBuilder>` element. It was used in the old `EditPage` and will probably be removed soon*.